cmd/program/program.go

```diff
@@ -216,7 +216,7 @@ func (p *Project) CreateMainFile() error {
 	// check if AbsolutePath exists
 	if _, err := os.Stat(p.AbsolutePath); os.IsNotExist(err) {
 		// create directory
-		if err := os.Mkdir(p.AbsolutePath, 0754); err != nil {
+		if err := os.Mkdir(p.AbsolutePath, 0o754); err != nil {
```

Comment:
What is this doing?

Reply:
This is nothing to do with the PR itself, go lsp in neovim auto formats octals 0o.